linux_os/guide/services/ntp/service_timesyncd_enabled/rule.yml

```diff
@@ -1,5 +1,7 @@
 documentation_complete: true
 
+prodtype: multi_platform_ubuntu
```

Comment:
you can't use multi_platform_* in prodtype, that's for platform elements in OVAL. You must enumerate ubuntu1404,ubuntu1604,ubuntu1804.

Reply:
Aha, so that is why the build failed, thanks.